src/mk.sh

```diff
@@ -30,6 +30,9 @@ elif [ $OSNAME = "Serenity" ]; then
   fi
   SDL2=liblsdl2
   GUI=linux
+elif [ $OSNAME = "Darwin" ]; then
```

Comment:
This allows the Xcode project to call this script to clean up build intermediates created by the makefiles.